templates/payments/workshop_detail.html

```diff
@@ -60,7 +60,13 @@
           {% endfor %}
         </tbody>
       </table>
-      <input type="submit" class="btn btn-primary pull-right" value="Continue" />
+      <div class="col-md-3 col-md-offset-9">
+          {% if workshop.public %}
+            {{ form.special_code }}
```

Comment:
Is this leftover from the earlier version where the discount code was manually entered? If so, can you remove this?